examples/dumpers.R

```diff
@@ -0,0 +1,38 @@
+\dontrun{
```

Comment:
@mbojan Can you put this file in a folder called `man-roxygen` like this https://github.com/ropensci/fulltext/tree/master/man-roxygen - then `.Rbuildignore` `man-roxygen` instead of `examples`